docs/designers-developers/developers/block-api/block-attributes.md

```diff
@@ -2,13 +2,18 @@
 
 ## Common Sources
 
-Attribute sources are used to define the strategy by which block attribute values are extracted from saved post content. They provide a mechanism to map from the saved markup to a JavaScript representation of a block.
+Attribute sources are used to define how the block attribute values are extracted from saved post content. They provide a mechanism to map from the saved markup to a JavaScript representation of a block.
 
 If no attribute source is specified, the attribute will be saved to (and read from) the block's [comment delimiter](/docs/designers-developers/key-concepts.md#delimiters-and-parsing-expression-grammar).
 
-Each source accepts an optional selector as the first argument. If a selector is specified, the source behavior will be run against the corresponding element(s) contained within the block. Otherwise it will be run against the block's root node.
+The keys specified in the attributes source object are named as you see fit. The result of the attribute source definition will be assigned as a value to each key.
```

Comment:
Should we to "The result of the attribute source definition will be assigned as a value to each key." to "The result of the attribute source definition is assigned as a value to each key." as it looks even more direct and in accordance with the rest of the text e.g: the source definition runs against ...

Reply:
:+1: